.github/workflows/ci.yml

```diff
@@ -0,0 +1,35 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - 'master'
+  push:
+    branches:
+      - '*'
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
Why a schedule? 😸

Reply:
It's just something I've gotten used to. Plus, it gives me a daily reminder if a test is failing to get it fixed. We can remove it if you prefer, I just took the same yml from another setup and tweaked it a bit

Reply:
Just out of curiosity. Try to prevent nighlty builds, but in your case its just “convenience” which is fine. So fine for me